src/plugins/action_trim_responses/src/trimResponses.ts

```diff
@@ -0,0 +1,70 @@
+import databaseMethods from './databaseMethods'
+import isEqual from '@openmsupply/deep-comparison'
```

Comment:
is this using https://www.npmjs.com/package/deep-equal ?

Reply:
Nah it's one I wrote myself ages ago for dealing with plugin loading (I think). It's currently in the `utilityFunctions.ts` file (and has a pretty thorough test suite), but I built it into a private package (in this repo) so we could use it in plugins. That cool?